markdown/solace-helm-environment-setup/helm-environment-setup.md

```diff
@@ -13,100 +13,174 @@ feedback link: https://github.com/SolaceDev/solace-dev-codelabs
 
 [https://helm.sh/docs/intro/install/](https://helm.sh/docs/intro/install/)
 
+Open a Ubuntu Window and enter the following commands.
+
 ```bash
 $ wget https://get.helm.sh/helm-v3.2.0-linux-amd64.tar.gz
 $ tar -zxvf helm-v3.2.0-linux-amd64.tar.gz
 $ sudo mv linux-amd64/helm /usr/local/bin/helm
 ```
 
+![Helm Install](./img/Step-3-1-Helm-Setup.png)
+
+### Step 3.2:  Install Help Repos
+
 Finally install the Helm repos:
 
 ```bash
-$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$ helm repo add stable https://charts.helm.sh/stable
 $ helm repo add solacecharts https://solaceproducts.github.io/pubsubplus-kubernetes-quickstart/helm-charts
 ```
 
-If you click on the Kubernetes extension in VSC, you should see the Helm repos.
+![Helm Repos](./img/Step-3-2-Helm-Repos.png)
+
+It is also possible to install Helm in Windows if you prefer to work in PowerShell.  If you use Chocolatey, there is a [Helm package](https://community.chocolatey.org/packages/kubernetes-helm) available there.
+
+### Next step is to configure the extensions in VSC.
+
+## Step 4: Install Visual Studio Code Extensions
 
-![VSC - Helm repos](./img/image009.png)
+Duration: 0:02:00
+
+Install [Visual Studio Code](https://code.visualstudio.com/Download) (if not already installed).
+
+### Step 4.1:  Install the following extensions in Visual Studio Code:
+
+- **Remote - Development Pack** – This will allow you to use the Ubuntu terminal in Code (instead of Power Shell)
+- **Kubernetes** – This extension allows you to install Helm charts directly from Code to different Kubernetes backends (in this case Docker Desktop)
+
+Note:  I also installed the Docker extension but, this is not required.
+
+With Code and the Remote – WSL extension, everything else can be done from the Ubuntu terminal in Code. 
+
+### Step 4.2:  Launch Remote VSC Window
+
+Launch VSC remote window.  Use the "Remote-WSL: New Window Using Distro..." command to open a remote window to Ubuntu WSL.
+
+![VSC Remote Window](./img/Step-4-1-VSC-Remote-Distro.png)
+
+The new remote window has "WSL: Unbuntu" in the bottom left corner to indicate the environment that the remote VSC window is related to.
+
+![VSC - Helm repos](./img/Step-4-2-VSC-Remote-Ubuntu.png)
+
+Note: Windows Firewall may ask you to allow access for node setup the VSC remote server.
+
+### Step 4.3:  Install the Kubernetes Extension in the Remote WSL: Ubuntu Window
+
+Install the Kubernetes extension in the remote VSC window (same as previously done in the local VSC window).
 
 With that the environment is setup, the Kubernetes extension can see the cluster running in Docker Desktop and the PubSub+ charts in the Helm Hub. Now we are ready to deploy a chart.
 
+![VSC - Helm repos](./img/Step-4-3-VSC-Helm-Repos.png)
+
+### The next step is to depoy the PubSub+ Helm chart
+
 ## Step 5: Install the PubSub+ Helm Chart
 
```

Comment:
Hey @paul-kondrat can you please add a `Duration: 0:0X:00` section under the ## header for step 5 so that it is reflected in the overall duration of the codelab. I believe this section would take 2 minutes so just add Duration: 0:02:00 here. after you do , run the following
```
./export.sh
cd ../../
git commit -a -m "added duration under section 5"
```

Reply:
Added the duration.